### ISSUE TITLE
New project - webpack error

### ISSUE THREAD

**Issue by stevesloka** [gobuffalo/buffalo#310]
My first time using buffalo, I installed the binary, then did a buffalo new to build a project. When building or trying to run dev, I get the following error, I do have node and webpack installed, anything else I can check?

```
Buffalo version 0.7.4

--> cleaning up target dir
--> running node_modules/.bin/webpack
Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema.
 - configuration.output.path: The provided value "./public/assets" is not an absolute path!
--> cleaning up build
----> cleaning up buffalo_build_main.go
Usage:
  buffalo build [flags]

Aliases:
  build, b, bill


Flags:
  -e, --extract-assets   extract the assets and put them in a distinct archive
  -o, --output string    set the name of the binary (default "bin/clair-ui")
  -z, --zip              zips the assets to the binary, this requires zip installed

Error: exit status 1

```

**Comment by markbates** [comment]
Web pack released their version 2.3.0 which broke compatibility with 2.2.x. The all of a sudden declared that that needs to be an absolute path.

The quickest fix is to change line 13 in `webpack.config.js` to this:

```js
    path: __dirname + "/public/assets"
```

**Comment by markbates** [comment]
you're welcome. i released `0.7.4.1` to fix the generated file, but i'm a little frustrated they broke it. :(
